source/nodejs/adaptivecards/src/carousel.ts

```diff
@@ -268,20 +314,23 @@ export class Carousel extends Container {
             "swiper-button-prev",
             "ac-carousel-left"
         );
+        prevElementDiv.style.zIndex = "20";
```

Comment:
Can you add a comment on it?
Also will it only applies to design mode?

Reply:
Will do. I'll also update so it is only applied in design mode